.gitignore

```diff
@@ -4,3 +4,6 @@ examples/var/logs/*
 !examples/var/logs/.gitkeep
 
 /.idea
+
+/docker
+/docker-compose.yml
```

Comment:
could you please remove those together with `/.idea` (I missed it). That kind of things should go to [global .gitignore](https://docs.github.com/en/github/getting-started-with-github/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)

Reply:
Done :-)